winbinder/winbinder.php

```diff
@@ -0,0 +1,2076 @@
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $items
+ * @param $clear
+ * @param $param
+ *
+ * @return int
+ */
+function wbtemp_create_statusbar_items($ctrl, $items, $clear, $param)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param null $item - item can also be WBC_RTF_TEXT to get the RTF code of an RTF input
+ *
+ * @return int
+ */
+function wbtemp_get_text($ctrl, $item = null)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $text
+ * @param $item
+ *
+ * @return int
+ */
+function wbtemp_set_text($ctrl, $text, $item)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $selitems
+ *
+ * @return int
+ */
+function wbtemp_select_tab($ctrl, $selitems)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $value
+ * @param $item
+ *
+ * @return int
+ */
+function wbtemp_set_value($ctrl, $value, $item = null)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $item
+ * @param $image
+ * @param $value
+ *
+ * @return int
+ */
+function wbtemp_create_listview_item($ctrl, $item, $image, $value)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $index
+ * @param $value
+ *
+ * @return int
+ */
+function wbtemp_set_listview_item_checked($ctrl, $index, $value)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * Return TRUE if the item's checkbox is checked
+ * @param $ctrl
+ * @param $item
+ * @return bool
+ */
+function wbtemp_get_listview_item_checked($ctrl, $item)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $item
+ * @param $subitem
+ * @param $text
+ *
+ * @return int
+ */
+function wbtemp_set_listview_item_text($ctrl, $item, $subitem, $text)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $item
+ *
+ * @return mixed
+ */
+function wbtemp_get_listview_text($ctrl, $item)
+{
+    return '' | 0 | null | [];
+}
+
+/**
+ * Get the number of columns in the pwbo control,
+ *
+ * @param $ctrl
+ *
+ * @return int
+ */
+function wbtemp_get_listview_columns($ctrl)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $i
+ * @param $text
+ * @param $width - If nWidth is negative, calculate width automatically
+ * @param $align - WBC_LEFT, WBC_RIGHT, WBC_CENTER
+ *
+ * @return int
+ */
+function wbtemp_create_listview_column($ctrl, $i, $text, $width, $align)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ *
+ * @return int
+ */
+function wbtemp_clear_listview_columns($ctrl)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $item
+ * @param $selected
+ *
+ * @return int
+ */
+function wbtemp_select_listview_item($ctrl, $item, $selected)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $ctrl
+ * @param $bool
+ *
+ * @return int
+ */
+function wbtemp_select_all_listview_items($ctrl, $bool)
+{
+    return '' | 0 | null;
+}
+
+/**
+ * @param $parent
+ * @param $caption
+ *
+ * @return int
+ */
+function wbtemp_create_menu($parent, $caption)
```

Comment:
https://github.com/wagy/WinBinder/blob/4fdc83f93006bb9de89435b8ec212fe851598431/phpwb_export.c#L32-L39 should we have this function here?